CODEOWNERS

```diff
@@ -2,11 +2,13 @@
 *                   @alisw/infrastructure-admins
 
 # Dataflow team
+coconut             @alisw/dataflow-experts
 common-o2*          @alisw/dataflow-experts
 configuration*      @alisw/dataflow-experts
 control*            @alisw/dataflow-experts
 datasampling*       @alisw/dataflow-experts
 flpproto*           @alisw/dataflow-experts
+golang              @alisw/dataflow-experts
```

Comment:
Please assign the ownership of `golang` to `@alisw/infrastructure-admins`, thanks!